templates/tfengine/components/cicd/triggers/variables.tf

```diff
@@ -112,4 +112,5 @@ variable "terraform_root_prefix" {
 variable "service_account_email" {
   type = string
   description = "Email of the Cloud Scheduler service account."
+  default = ""
```

Comment:
Why is this optional?

Reply:
It's because the block that defines the resource that fills the variable is optional:

```
resource "google_service_account" "cloudbuild_scheduler_sa" {
  project      = var.project_id
  account_id   = "cloudbuild-scheduler-sa"
  display_name = "Cloud Build scheduler service account"
  depends_on = [
    google_project_service.services,
  ]
}
```

It only appears if there are scheduled jobs. Now it's needed to conditionally render this variable, son when it's not there, the default it's empty to avoid the error, but it's not really needed since it's not used.